metaci/release/models.py

```diff
@@ -135,6 +154,13 @@ class Release(StatusModel):
         null=False,
         blank=False,
     )
+    release_cohort = models.ForeignKey(
+        "release.ReleaseCohort",
+        on_delete=models.PROTECT,
+        null=True,
+        blank=False,
```

Comment:
I think blank also needs to be set to True so that it's possible to edit the Release in the admin without setting a release cohort.